integrations/elasticsearch/src/haystack_integrations/document_stores/elasticsearch/document_store.py

```diff
@@ -106,8 +106,20 @@ def __init__(
         # configure mapping for the embedding field
         mappings = {
             "properties": {
-                "embedding": {"type": "dense_vector", "index": True, "similarity": embedding_similarity_function}
-            }
+                "embedding": {"type": "dense_vector", "index": True, "similarity": embedding_similarity_function},
+                "content": {"type": "text"},
+            },
+            "dynamic_templates": [
+                {
+                    "strings": {
+                        "path_match": "*",
```

Comment:
Nit: not sure you need this line, the template will run on any string field anyways

Reply:
Yes, that's correct. It's not needed and I removed it now. Thank you!